src/main/java/com/icthh/xm/ms/scheduler/domain/enumeration/StateKey.java

```diff
@@ -0,0 +1,8 @@
+package com.icthh.xm.ms.scheduler.domain.enumeration;
+
+/**
+ * The StateKey enumeration.
+ */
+public enum StateKey {
+    ACTIVE, DONE
```

Comment:
@vkirichenko @ishkurko 
do we need to distinguish between DONE task and Expired?

Reply:
in future I think yes, we need to discuss life cycle and create separate task
in current version i suggest leave this states